src/hacbs/pages/PipelineRunPage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { PageSection, PageSectionVariants } from '@patternfly/react-core';
+import NamespacedPage from '../../components/NamespacedPage/NamespacedPage';
+import PageLayout from '../../components/PageLayout/PageLayout';
+import { useQuickstartCloseOnUnmount } from '../../hooks/useQuickstartCloseOnUnmount';
+import { getQueryArgument } from '../../shared/utils';
+import PipelineDetailsView from '../components/PipelineRunDetailsView/PipelineRunDetailsView';
```

Comment:
nit: use file name as component name
```suggestion
import PipelineRunDetailsView from '../components/PipelineRunDetailsView/PipelineRunDetailsView';
```

Reply:
👍